format-definitions/parsec-key-evidence-tpm.md

```diff
@@ -0,0 +1,82 @@
+# Parsec TPM key attestation data format
+
+This document describes the data encoding for key attestation tokens produced
+using a TPM. The format relies on the [Attestation
+Object](https://www.w3.org/TR/webauthn/#sctn-attestation) format defined in
+WebAuthN, using the [TPM attestation statement
+format](https://www.w3.org/TR/webauthn-2/#sctn-tpm-attestation). All data
+structures below are expected to be encoded in [CTAP canonical CBOR
+encoding](https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html#ctap2-canonical-cbor-encoding-form).
+
+## TPM Attestation Statement Format
+
+The attestation statement format defined for TPM backends is defined below.
+
+CDDL representation of Parsec TPM Key Attestation Statement Format:
+
+```
+parsecTpmKeyStmtFormat = {
+                 tpmVer: "2.0",
+                 alg: COSEAlgorithmIdentifier,
```

Comment:
As discussed with @ionut-arm we do not need to encode the alg, separately in the "key" statement as the algorithm is already provided in the sig bytes encoded by TPM